documents/operating.md

```diff
@@ -0,0 +1,329 @@
+the President of BIFFUD and the President of the United States.
+
+Corporate Overlords can also vote to remove a peer from the organization if
+any of the following conditions apply:
+
+- The Corporate Overlord fails to attend 50% of the monthly plotting sessions
+in a given year without a note from a family member explaining its absence.
+- The Corporate Overlord has violated a serious federal law in the past year.
+- The Corporate Overlord is hated or despised.
+
+The removal of a corporate overlord can be nominated during a plotting session
+and requires 70% approval or more from the other Corporate Overlords.
+
+Overlords are entitled to a pro-rated profit share for their tenure in a given
+year, as well as ongoing profit-share according to their ownership of various
+BIFFUD projects.
+
+---------------------------------------------------------------------
+SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
+---------------------------------------------------------------------
+- [x] Types of project
+- [x] Evaluating ideas
+- [x] IP (Intellectual Property / Internet Protocol)
+- [x] Open source
+
+### Types of Project
+A. **Grant Funded:** projects funded by grants from third parties are executed
+by the BIFFUD member who has led the relationship with the funders, or by an
+alternative BIFFUD member as delegated by that applicant.  Money associated with
+a grant is to be distributed as per the grant's agreement.
+
+B. **Member Owned:** projects that are led or adopted by BIFFUD members.  The
+owners of a project must all be BIFFUD members, and the percentages and specific
+terms of ownership should be define by those members as part of their application
+for the adoption of the idea under BIFFUD.
+
+C. **BIFFUD Core:** projects that are taken on by the entire corporation.  These
+might be projects utilized by the Corporate Overlords to execute on company policy,
+or simply be projects that the Corporate Overlords decide to develop without a concept
+of individual ownership.
+
+
+### Evaluating Ideas
+Anybody can build anything they want at any time, but in order for something to become
+an official BIFFUD project it must be formally adopted by the Corporate Overlords during
+a plotting session.  The determination of what makes a project a BIFFUD project is
+ultimately determined by that process, but in the spirit of framing the conversation,
+the following questions are recommended.
+
+1. How is this project a bad idea?
+2. Does it make you thinking face emoji?
+3. Is it realistic to implement through BIFFUD?
+4. If the project were a character in Dungeons and Dragons, what would it's alignment be?
+
+The BIFFUD brand deserves compelling answers to questions 1 and 2. Projects should be
+realistic to implement, and should either be "Chaotic Good", "Neutral Good", or
+"Chaotic Neutral".  All of these data points simply serve as guidance. The ultimate
+decision of what projects should be adopted falls on the Corporate Overlords.
+
```

Comment:
Should explain what it means to be a BIFFUD project -- specifically talk about how at that point it becomes brought into the org's github, added to whatever public list / materials exist for (e.g. our website), and given a biffud subdomain.